.github/actions/setup-env-vars/action.yml

```diff
@@ -10,9 +10,10 @@ runs:
         SHA: ${{ github.sha }}
         branch: ${{ github.ref }}
         git_repo: ${{ github.repository }}
-        short_sha: $(git rev-parse --short "${{ github.sha }}")
         pr_number: ""
       run: |
+        
+        short_sha=$(git rev-parse --short "$SHA")
```

Comment:
Just curious, what's causing it to fail now?

Reply:
The reason is that `short_sha` depends on git command to shrink the hash. However, it doesn't work in the same way when it is defined under the `env`. Hence, moving it to runnable command should resolve the issue